### PR TITLE
Descriptions and unit tests - part 2

### PR DESCRIPTION
**Pull request by irina060981**
* Added descriptions for all other components
* very tiny refactor of tokenizeController
* some renamings and small checks

@balmas , I would only you to approvals list, because there is no any big or serious changes.

### CODE REVIEW

File: src/lib/data/alignment.js [hunk 1]
```diff
@@ -78,49 +116,99 @@ export default class Alignment {
...
   shouldStartNewAlignmentGroup () {
-    return !this.activeAlignmentGroup
+    return !this.hasActiveAlignment
+  }
+
+  /**
+   * Defines if there is an active alignment group
+   * @returns {Boolean}
+   */
+  get hasActiveAlignment () {
```
Comment (balmas) [inline src/lib/data/alignment.js:165]: I still see direct references to this.activeAlignmentGroup elsewhere. Did you mean to update them to use this method instead?

File: src/lib/data/alignment.js [hunk 2]
```diff
@@ -131,13 +219,23 @@ export default class Alignment {
     return false
   }
 
+  /**
+   * Finds Alignment Group by token, if token is grouped in some alignment group
+   * @param {Token} token
+   * @returns {AlignmentGroup | false}
```
Comment (balmas) [inline src/lib/data/alignment.js:225]: Alter the return type based upon whether or not it is found seems a little odd. Isn't it more usual to return null?

File: src/lib/l10n/l10n-singleton.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* istanbul ignore file */
```
Comment (balmas) [inline src/lib/l10n/l10n-singleton.js:1]: what is this comment?

**Review by balmas — COMMENTED** [review]
@irina060981 I had a couple of small comments/questions on the code changes. I haven't had a chance to review the tests in detail yet, but I'm fine with you merging without that (after responding to the code comments) and I'll review the tests as soon as I can. Thanks.

File: src/lib/data/alignment.js (hunk 1 above)
Comment (irina060981) [reply src/lib/data/alignment.js:165]: Yes, I missed it - thank you for the catch! Fixed

File: src/lib/data/alignment.js (hunk 2 above)
Comment (irina060981) [reply src/lib/data/alignment.js:225]: Ok - fixed

File: src/lib/l10n/l10n-singleton.js (hunk 3 above)
Comment (irina060981) [reply src/lib/l10n/l10n-singleton.js:1]: This comment removes this file from jest coverage - I removed L10n files from coverage - as it won't be tested

**Comment by irina060981** [comment]
> I had a couple of small comments/questions on the code changes. I haven't had a chance to review the tests in detail yet, but I'm fine with you merging without that (after responding to the code comments) and I'll review the tests as soon as I can. Thanks.

Ok, I will merge it - thank you :)

**Pull request merged**